### PR TITLE
Switch to Gradle

### LINKED ISSUE #4: Convert to Gradle and make this work with jME 3.6.1
Convert to Gradle and make this work with jME 3.6.1. Yeah, an example in the official repo. I guess the least to do is to just try to restore this a bit.

The problem being currently that nobody knows with what this project is supposed to be opened with. It links jME locally. It is probably old SDK Ant format. It just doesn't build anymore.

With Gradle it works regardless what you open it with.

### PR DESCRIPTION
**Pull request by tonihele**
Changes the whole build from Ant to Gradle. The Gradle build is 100% compatible to what jME SDK expects a Gradle project to look like.

It seems to work out of the box with the latest jME 3.6.1 and Java 17. So I also set them in.

Resolves #4

### CODE REVIEW

File: Skulls/build.gradle [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+group 'com.jme3'
+version '1.0'
+
+mainClassName = "com.jme3.skulls.MainApplication"
+
+repositories {
+    mavenCentral()
+    jcenter()
```
Comment (MeFisto94) [inline Skulls/build.gradle]: This should at least be a deprecation warning if not straight not working, right? Or is jcenter still in this read-only archived state?

File: Skulls/build.gradle [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    maven { url 'https://jitpack.io' }
```
Comment (MeFisto94) [inline Skulls/build.gradle]: I see no com.github dependency, so do we need jitpack?

**Review by MeFisto94 — APPROVED** [review]
One thing we actually loose is the launch4j and android solutions. It's not related to this PR, but something we may craft a standard usecase for given some time, though.

**Comment by tonihele** [comment]
I removed the extra repos

**Pull request merged**